src/World.h

```diff
@@ -79,7 +79,7 @@ typedef cItemCallback<cFlowerPotEntity>    cFlowerPotCallback;
 
 
 
-
+const int CHUNK_CHANCE_ACCURACY = 100000;
```

Comment:
Just an idea, but I'm wondering:

1. What are the chances that realistically anyone notices and
2. How does that compare with the complexity of the rest of these changes?